ee/rpc/sior/src/sior_rpc.c

```diff
@@ -26,7 +26,7 @@
 
 static SifRpcDataQueue_t qd __attribute__((aligned(64)));
 static SifRpcServerData_t Sd0 __attribute__((aligned(64)));
-static u32 buffer[32] __attribute__((aligned(64)));
+static u32 buffer[64] __attribute__((__aligned__(64)));
```

Comment:
128 byte to 256 byte ?

Reply:
Ah whoops, this was an unintended change.